docker-compose_dev.yml

```diff
@@ -32,30 +33,30 @@ services:
     networks:
       - otenet
 
-  sftp:
-    image: atmoz/sftp
-    volumes:
-      - sftp-storage:${HOME:-/home/foo}/download
-    command: ${USER:-foo}:${PASSWORD:-pass}:1001
-    networks:
-      - otenet
+#  sftp:
```

Comment:
Remove in separate PR - create issue for it

Reply:
https://github.com/EMMC-ASBL/oteapi-services/issues/397